### PR TITLE
fixed sprint changes bug

### PR DESCRIPTION
**Pull request by shibangi23**
### Functionality:
On coming back from task page the selected team ID and sprint  number are maintained.

### Solution:
Used applicationSettingService to maintain the state when we select a sprint in filter. 

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start
filter sprint
come back

### CODE REVIEW

**Review by arjund-06 — APPROVED** [review]
ship it

**Review by arjund-06 — COMMENTED** [review]
Please resolve merge conflicts

**Comment by shibangi23** [comment]
Resolved merge conflicts. Please review once.

**Review by arjund-06 — APPROVED** [review]
ship it

File: src/app/services/applicationSettings/application-settings.service.ts [hunk 1]
```diff
@@ -8,11 +8,15 @@ import { BackendService } from '../backend/backend.service';
 @Injectable({
   providedIn: 'root'
 })
+
 export class ApplicationSettingsService {
+  public edited: boolean = false;
```
Comment (viv9k) [inline src/app/services/applicationSettings/application-settings.service.ts:13]: What does edited means here. This doesn't clarify the right meaning of the variable.

We need to specify the right name, also we don't need this variable.

Rather you can directly have variables like selectedTeamId and selectedSprintId.

And change it in application service, whenever needed.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fix

File: src/app/services/applicationSettings/application-settings.service.ts (hunk 1 above)
Comment (shibangi23) [reply src/app/services/applicationSettings/application-settings.service.ts:13]: "edited" variable is used to check if the sprint was edited after the application was launched or not.

Comment (viv9k) [reply src/app/services/applicationSettings/application-settings.service.ts:13]: You can check it by comparing old and new values.

If they are not equal then they are edited.
else not edited.

**Review by viv9k — CHANGES_REQUESTED** [review]
Now, check it again

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**